validators/src/unit-test/java/uk/gov/moj/sdt/validators/BulkFeedbackRequestValidatorTest.java

```diff
@@ -124,47 +129,38 @@ public class BulkFeedbackRequestValidatorTest extends AbstractValidatorUnitTest
     /**
      * Setup of the Validator and Domain class instance.
      */
+    @BeforeEach

```

Comment:
I don't think the BeforeEach annotation is needed.  The setupLocalTests() method is invoked from setUp() on AbstractSdtUnitTestBase which AbstractValidatorUnitTest inherits from.  The setUp() method is annotated with BeforeEach so it's not needed here.

Reply:
Removed